wallet/txauthor/author.go

```diff
@@ -133,14 +135,12 @@ func NewUnsignedTransaction(outputs []*wire.TxOut, relayFeePerKb dcrutil.Amount,
 			changeIndex = l
 		}
 
-		estSignedSize := txsizes.EstimateSerializeSize(scriptSizers,
```

Comment:
Keep this.  If change can't be added because it produces dust, we must return the estimate without any change added, which is why this used the actual transaction outputs with no (possibly additional) added change.